modules/apps/ccv/child/keeper/relay_test.go

```diff
@@ -22,56 +23,95 @@ func (suite *KeeperTestSuite) TestOnRecvPacket() {
 	suite.Require().NoError(err)
 	pk2, err := cryptocodec.ToTmProtoPublicKey(ed25519.GenPrivKey().PubKey())
 	suite.Require().NoError(err)
+	pk3, err := cryptocodec.ToTmProtoPublicKey(ed25519.GenPrivKey().PubKey())
+	suite.Require().NoError(err)
 
-	pd := types.NewValidatorSetChangePacketData(
-		[]abci.ValidatorUpdate{
-			{
-				PubKey: pk1,
-				Power:  30,
-			},
-			{
-				PubKey: pk2,
-				Power:  20,
-			},
+	changes1 := []abci.ValidatorUpdate{
+		{
+			PubKey: pk1,
+			Power:  30,
+		},
+		{
+			PubKey: pk2,
+			Power:  20,
+		},
+	}
+
+	changes2 := []abci.ValidatorUpdate{
+		{
+			PubKey: pk2,
+			Power:  40,
+		},
+		{
+			PubKey: pk3,
+			Power:  10,
 		},
+	}
+
+	pd := types.NewValidatorSetChangePacketData(
+		changes1,
 	)
 
-	packet := channeltypes.NewPacket(pd.GetBytes(), 1, parenttypes.PortID, suite.path.EndpointB.ChannelID, childtypes.PortID, suite.path.EndpointA.ChannelID,
-		clienttypes.NewHeight(1, 0), 0)
+	pd2 := types.NewValidatorSetChangePacketData(
+		changes2,
+	)
 
 	testCases := []struct {
-		name           string
-		malleatePacket func()
-		expErrorAck    bool
+		name                   string
+		packet                 channeltypes.Packet
+		newChanges             types.ValidatorSetChangePacketData
+		expectedPendingChanges types.ValidatorSetChangePacketData
```

Comment:
```suggestion
		expPendingChanges types.ValidatorSetChangePacketData
```
nit: naming consistency